### PR TITLE
feat(keymaps): Changes to crkbd/keymaps/armand1m layout and README.md update

### PR DESCRIPTION
**Pull request by armand1m**
## Description
Changes to crkbd/keymaps/armand1m layout and README.md update. 
Also adds JSON files for Keyboard Layout Editor.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json [hunk 1]
```diff
@@ -0,0 +1,281 @@
+[
```
Comment (drashna) [inline keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json:1]: Could you remove this?

File: keyboards/crkbd/keymaps/armand1m/keymap.c [hunk 2]
```diff
@@ -19,48 +19,51 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-      _______,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                         KC_6,    KC_7,    KC_8,    KC_9,    KC_0, _______,
-  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, XXXXXXX, _______,
-  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______,
-  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                         _______, _______,  _______,    _______, _______, _______
-                                      //`--------------------------'  '--------------------------´
+      KC_TRANSPARENT,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                       KC_6,    KC_7,    KC_8,      KC_9,    KC_0, KC_TRANSPARENT,
```
Comment (drashna) [inline keyboards/crkbd/keymaps/armand1m/keymap.c:49]:
```suggestion
      _______,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                       KC_6,    KC_7,    KC_8,      KC_9,    KC_0, _______,
```
And the rest could be changed back as well.

Same with `XXXXXXX` -> `KC_NO`

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json (hunk 1 above)
Comment (armand1m) [reply keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json:1]: I'd like to be able to keep it so I can import it back into the keyboard layout editor and update this on any layout changes. Do you feel it is unnecessary to keep it versioned?

Comment (fauxpark) [reply keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json:1]: We generally don't allow KLE JSON files in the repo, it adds bloat for no good reason as nothing in QMK uses it.

File: keyboards/crkbd/keymaps/armand1m/keymap.c [hunk 3]
```diff
@@ -19,48 +19,51 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+      case ARROW_FUNCTION:
+        if (record->event.pressed) {
+            SEND_STRING("() => {}");
+        } else {
+            SEND_STRING(SS_TAP(X_LEFT) SS_TAP(X_ENTER)); 
+        }
+        break;
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/armand1m/keymap.c:34]:
```suggestion
        return false;
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json (hunk 1 above)
Comment (armand1m) [reply keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json:1]: Thanks for the explanation! I'll take it out then. I'll store then on some other repo and just link in the README instead

Comment (armand1m) [reply keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json]: Removed all KLE JSON files

**Review by fauxpark — APPROVED** [review]

File: keyboards/crkbd/keymaps/armand1m/keymap.c [hunk 4]
```diff
@@ -19,48 +19,51 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+      _______,   KC_EXLM,     KC_AT,   KC_HASH,         KC_DLR, KC_PERC,             KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, _______,
+      _______,   XXXXXXX,   XXXXXXX,   XXXXXXX, ARROW_FUNCTION,   KC_LT,             KC_MINS,  KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS,  KC_GRV,
+      _______,   XXXXXXX,   XXXXXXX,   XXXXXXX,        XXXXXXX,   KC_GT,             KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE, KC_TILD,
+                                                  _______, _______, _______,     _______, _______, _______
   )
 };
 
 #ifdef OLED_DRIVER_ENABLE
 oled_rotation_t oled_init_user(oled_rotation_t rotation) {
   if (!is_master) {
```
Comment (drashna) [inline keyboards/crkbd/keymaps/armand1m/keymap.c:65]: ideally, the `is_master` should be replaced with `is_keyboard_master()`, but that may not work on legacy.  But hopefully, legacy will be deprecated in the future.

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/armand1m/keymap.c (hunk 4 above)
Comment (armand1m) [reply keyboards/crkbd/keymaps/armand1m/keymap.c:65]: @drashna thanks for the heads up! I'll test the change and consider it in a future pull request. Will also add a note about legacy compatibility on the README if I end up doing so

**Pull request merged**